### PR TITLE
Please add vc5 to emerging applications list

### PR DESCRIPTION
**Pull request by davidcoles**
Project is eBPF/XDP based, open source (GPL2), uses a founder-leader model, under active development and has been used in production for over a year.

Many thanks.

### CODE REVIEW

File: src/data/pages/applications/emerging.js [hunk 1]
```diff
@@ -479,6 +479,18 @@ const emergingApplications = [
...
     ],
   },
+  {
+    logoUrl: 'https://github.com/davidcoles/vc5',
+    name: 'vc5',
+    logoName: 'vc5Logo',
+    title: 'Layer 4 load balancer for Linux',
+    description: `vc5 is an XDP based Direct Server Return (DSR) layer 4 laod balancer with a use case in replacing
+    hardware appliances. Service health checks and BGP-based Route Health Injection (RHI) are built in. A web console,
+    JSON stats endpoint, Prometheus metrics, logging with Elasticsearch and alerting with webhooks are included.`,
```
Comment (xmulligan) [inline src/data/pages/applications/emerging.js:489]:
```suggestion
    JSON stats endpoint, Prometheus metrics, logging with Elasticsearch, and alerting with webhooks are included.`,
```

**Comment by xmulligan** [comment]
Thanks for the addition, just one small nit

**Comment by davidcoles** [comment]
> Thanks for the addition, just one small nit

Well, you did go to Oxford after all!

**Review by xmulligan — APPROVED** [review]

**Comment by xmulligan** [comment]
Exactly ;) thanks for the addition

**Pull request merged**